scipy/interpolate/_fitpack_py.py

```diff
@@ -564,6 +567,97 @@ def spalde(x, tck):
+    >>> m1_y_deriv1 = splev(x, bspl, der=1)  # 1st derivative
+    >>> m1_y_deriv2 = splev(x, bspl, der=2)  # 2nd derivative
+    >>> m1_y_deriv3 = splev(x, bspl, der=3)  # 3rd derivative
+    >>> # Method 2: splder() -----------------------------------------------------------
+    >>> m2_y_bsplin = splder(bspl, n=0)(x)  # The B-spline itself
+    >>> m2_y_deriv1 = splder(bspl, n=1)(x)  # 1st derivative
+    >>> m2_y_deriv2 = splder(bspl, n=2)(x)  # 2nd derivative
+    >>> m2_y_deriv3 = splder(bspl, n=3)(x)  # 3rd derivative
+    >>> # Method 3: spalde() -----------------------------------------------------------
+    >>> m3_y_bsplin = [spalde(i, tck)[0] for i in x ]  # The B-spline itself
+    >>> m3_y_deriv1 = [spalde(i, tck)[1] for i in x ]  # 1st derivative
+    >>> m3_y_deriv2 = [spalde(i, tck)[2] for i in x ]  # 2nd derivative
+    >>> m3_y_deriv3 = [spalde(i, tck)[3] for i in x ]  # 3rd derivative
+    
+    >>> # Graphic comparison ===========================================================
+    >>> fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(16, 5))
+    >>> fig.suptitle(f'Evaluate a B-spline and all derivatives')
+    >>> # Method 1: splev() ------------------------------------------------------------
+    >>> ax1.plot(x, m1_y_bsplin, label='B-spline')
+    >>> ax1.plot(x, m1_y_deriv1, '--', color='#999999', label='1st deriv.')
+    >>> ax1.plot(x, m1_y_deriv2, '-.', color='#999999', label='2nd deriv.')
+    >>> ax1.plot(x, m1_y_deriv3, ':', color='#999999', label='3rd deriv.')
+    >>> # Plot a reference line
+    >>> ax1.vlines(x_ref, -11, 10.0, color='r', label=f'x = {x_ref}')
+    >>> # Aditional info
+    >>> ax1.set_title('splev()')
+    >>> ax1.legend()
+    >>> # Method 2: splder() ------------------------------------------------------------
+    >>> ax2.plot(x, m2_y_bsplin, label='B-spline')
+    >>> ax2.plot(x, m2_y_deriv1, '--', color='#999999', label='1st deriv.')
+    >>> ax2.plot(x, m2_y_deriv2, '-.', color='#999999', label='2nd deriv.')
+    >>> ax2.plot(x, m2_y_deriv3, ':', color='#999999', label='3rd deriv.')
+    >>> # Plot a reference line
+    >>> ax2.vlines(x_ref, -11, 10.0, color='r', label=f'x = {x_ref}')
+    >>> # Aditional info
+    >>> ax2.set_title('splder()')
+    >>> ax2.legend()
+    >>> # Method 3: spalde() -----------------------------------------------------------
+    >>> ax3.plot(x, m3_y_bsplin, label='B-spline')
+    >>> ax3.plot(x, m3_y_deriv1, '--', color='#999999', label='1st deriv.')
+    >>> ax3.plot(x, m3_y_deriv2, '-.', color='#999999', label='2nd deriv.')
+    >>> ax3.plot(x, m3_y_deriv3, ':', color='#999999', label='3rd deriv.')
+    >>> # Plot a reference line
+    >>> ax3.vlines(x_ref, -11, 10.0, color='r', label=f'x = {x_ref}')
+    >>> # Aditional info
+    >>> ax3.set_title('spalde()')
+    >>> ax3.legend()
+    >>> # Plot -------------------------------------------------------------------------
+    >>> plt.show()
```

Comment:
I think this might fix the sizing
```suggestion
    >>> plt.tight_layout()
    >>> plt.show()
```